apparmor.d/groups/apps/discord-chrome-sandbox

```diff
@@ -16,8 +16,8 @@ include <tunables/global>
 profile discord-chrome-sandbox @{exec_path} {
   include <abstractions/base>
 
-  capability setgid,
-  capability setuid,
+  #capability setgid,
+  #capability setuid,
```

Comment:
Actually needed on some system, do not comment them.

Reply:
It wasn't required on debian testing (trixie), even though it has `kernel.unprivileged_userns_clone = 1`
I will test this again on stable to make sure, but I suspect it is no longer a requirement, especially since other electron apps do not seem to need this either (or it is missing)

Reply:
Could this be related to this feature?  https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces

I was able to use discord without allowing the two mentioned capabilities, even on systems that have `kernel.unprivileged_userns_clone = 1` but I noticed that my arch test systems also have `kernel.unprivileged_userns_apparmor_policy = 1` and `kernel.apparmor_restrict_unprivileged_unconfined = 0`
If I understand the linked article correctly, it seems the later kernel setting allows the use of unprivileged userns in the context of apparmor? I can't find any documentation about these settings unfortunately, but I suspect they influence the required capabilities.

In any case, I did not find any distribution that required `setuid` or `setgid` and aa-log never complained about them missing either. Maybe they are no longer needed after all?

Reply:
According to https://gitlab.com/apparmor/apparmor/-/wikis/unprivileged_userns_restriction the mentioned kernel parameters allow apparmor to block the use of unprivileged userns, unless the apparmor profile includes  `allow userns create`. That would explain why current distributions do not require the capabilities, since applications like discord that are confined by apparmor, will automatically fall back to their suid chrome-sandbox startup script. Do I understand this correctly? If so, what is the consequence for these apparmor profiles? Add userns permissions?

Reply:
The structure of recent electron based app might have changed. Also, most electron based app disable internal sandboxing.

The new userns restriction feature is handled differently, with the new `userns,` rule that is already part of `abstractions/common/electron`.

Let keep it that way, and I will investigate and update things later. Change could be made to the electron abstraction and applied everywhere.